plugins/openai/src/gpt.ts

```diff
@@ -165,7 +164,7 @@ function toOpenAiTool(tool: ToolDefinition): ChatCompletionTool {
   };
 }
 
-export function toOpenAiTextAndMedia(part: Part): ChatCompletionContentPart {
+export function toOpenAiTextAndMedia(part: Part, visual_detail_level: VisualDetailLevel): ChatCompletionContentPart {
```

Comment:
ideally all this plugin work would be in a separate PR, but I can let it pass for now